structuredevent-service-legacy/src/main/java/com/sequenceiq/cloudbreak/structuredevent/service/telemetry/converter/StructuredEventToClusterShapeConverter.java

```diff
@@ -80,4 +90,28 @@
 
         return cdpClusterShape;
     }
+
+    private String getClusterTemplateDetails(String blueprintJson) {
+        JSONObject jsonObject = new JSONObject();
+        if (!blueprintJson.isEmpty()) {
+            JSONObject jo = null;
```

Comment:
Declaring "jo" here seems unnecessary as it's not used outside of the try block. Instead the first line of the try block can be modified to "JSONObject jo = new JSONObject(blueprintJson);"